src/main/java/service/UserService.java

```diff
@@ -0,0 +1,23 @@
+package service;
+
+import model.User;
+import repositoriy.AllUsersRepository;
+import java.util.List;
+import java.util.Optional;
+
+public class UserService {
+
+    public static UserService userService = new UserService();
+    public AllUsersRepository allUsersRepository = AllUsersRepository.getInstance();
+
+    public UserService() {
+    }
+
+    public UserService(AllUsersRepository allUsersRepository) {
+        this.allUsersRepository = allUsersRepository;
+    }
+    public Optional<User> getUser(String login,String password){
+        List<User> allUsers = allUsersRepository.getAll();
```

Comment:
In future applications with database this filtering should be done in repository on DB level, not in service